modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -152,22 +193,57 @@ function inope_policy_review_form_submit($form, &$form_state) {
 
   // If 'I have reviewed this policy and approved it for use' is selected.
   if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
+
+    // Only administrators should be able to approve notes.
+    if (in_array('administrator', $account->roles)) {
```

Comment:
Is there some weird phrasing in the business problem that you pulled this from?  If so -- sorry.  Both staff and administrators should be able to say that they approve of the policy.  THey are not approving it for the company - but just saying that they approve of the policy for themselves.